### DISCUSSION TITLE
How to add menus in a correct way?

### DISCUSSION THREAD

**Discussion by rafa4mat** [WinEdt-Team/WinEdt-Support#83] [1.1 General Help and Discussion]
Hello, I want to insert many menus for diferent snippets.

For example, after the Insert Menu, in Options/Menu/... I added
![snap004053](https://github.com/user-attachments/assets/cf1aabbf-c160-430f-aa26-b88725bf5a8a)

But I receive a message of the system about privilegies (of cannot create a folder Snippet -I do it with a file manage program, outside WinEdt-), and don´t let me save modifications in the file "c:\Program Files\WinEdt Team\WinEdt 11\Menus\Snippets\SnippetInputFrechas.edt" .

Obtaining
![snap004052](https://github.com/user-attachments/assets/30033434-a545-4240-9f7c-afa0691d61a9)

The question is: How Should I do (recommended way) this kind of things for?, because I´m a little 'affraid' for privilegies messages. 

Is more recommendable make this kind of thing using icon toolbar?. But the MENU should be define in MainManu.ini, I think.

Are there a special folder site to do this kind of things?.

**Comment by Alex-Simonic** [comment]
Start the Configuration Wizard (Options Menu). There you will see %B and %b folders. All your customizations should be inside %b and you should not modify anything inside %B. No need for elevated privileges if you follow this guideline.
